iothub_client/src/iothubtransport_mqtt_common.c

```diff
@@ -93,6 +97,7 @@ typedef struct SYSTEM_PROPERTY_INFO_TAG
     size_t propLength;
 } SYSTEM_PROPERTY_INFO;
 
+//TODO: add diagnostic properties support for C2D messages
```

Comment:
diagnostic properties support for C2D messages is P2 task, we have backlog to track this task. How about just leave TODO here so people would know it is on purpose instead of a bug?

Reply:
I believe we will have trouble if this TODO tags get into the repo.
@jebrando once told me that our code is inspected by some group at Microsoft that explicitly hunts for TODO statements... 
he might be able to give more details.

Reply:
TODO removed.